netlify.toml

```diff
@@ -2,3 +2,13 @@
 	from = "/discord"
 	to = "https://discord.gg/6MZWP8z"
 	status = 301
+
```

Comment:
@TraceyHolinka, per the build log, I think we actually need to remove these newlines? https://app.netlify.com/sites/vuedc/deploys/5f66639b08ea9e0007a75a60 

> 4:01:42 PM: Invalid netlify.toml file: Error while decoding file /opt/build/repo/netlify.toml: Near line 7 (last key parsed 'redirects.from'): strings cannot contain newlines